src/main/resources/model/linked_data_authority.json

```diff
@@ -6,6 +6,11 @@
     "id": {
       "index": "keyword"
     },
+    "tenantId": {
+      "index": "keyword",
+      "searchTypes": [ "facet", "filter" ],
+      "showInResponse": [ "search" ]
+    },
```

Comment:
Will the `tenantId` be the field of incoming Kafka messages bodies?
We have an ability to get tenantId from Kafka headers, you can check other index mappings for tenantId field definition, if needed.

Reply:
Yes, it will. Took this definition from the Instance model initially. But I see your point: some other models contains the "isTenant": true". Added it ti LD models. Thank you!